### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by kordejong**
I fixed some small errors in the documentation (code references + text). Well, I think I fixed them. I am not a native English speaker.

### CODE REVIEW

File: docs/sphinx/examples/1d_stencil.rst [hunk 1]
```diff
@@ -82,7 +82,7 @@ string dataflows together and construct an execution tree.
 
 After the values of the futures in dataflow are ready, the values must be pulled
 out of the future container to be passed to the function ``heat``. In order to
-do this, we use the HPX facility ``unwrapped``, which underneath calls
+do this, we use the HPX facility ``unwrapping``, which underneath calls
```
Comment (hkaiser) [inline docs/sphinx/examples/1d_stencil.rst]: The facility is actually called `unwrapped`, so this is not a typo.

File: docs/sphinx/examples/1d_stencil.rst [hunk 2]
```diff
@@ -150,12 +150,12 @@ called unless the data from the neighboring partitions have arrived. Therefore
 it should come as no surprise that we synchronize this operation with another
 instance of dataflow (found in ``heat_part``). This dataflow is passed futures
 to the data in the current and surrounding partitions by calling ``get_data()``
-on each respective partition. When these futures are ready dataflow passes then
-to the ``unwrapped`` function, which extracts the shared_array of doubles and
+on each respective partition. When these futures are ready dataflow passes them
+to the ``unwrapping`` function, which extracts the shared_array of doubles and
```
Comment (hkaiser) [inline docs/sphinx/examples/1d_stencil.rst]: Same here, the HPX facility is called `unwrapped`.

Comment (kordejong) [reply docs/sphinx/examples/1d_stencil.rst]: I renamed `unwrapped` back to `unwrapping`.

File: docs/sphinx/examples/1d_stencil.rst (hunk 1 above)
Comment (kordejong) [reply docs/sphinx/examples/1d_stencil.rst]: I renamed `unwrapped` back to `unwrapping`.

**Pull request merged**